src/test/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/BalanceSheetTransformerTests.java

```diff
@@ -20,10 +14,18 @@
     private static final Long CURRENT_CALLED_UP_SHARE_CAPITAL = 1L;
     private static final Long CURRENT_TANGIBLE = 5L;
     private static final Long CURRENT_FIXED_ASSETS_TOTAL = 5L;
+    private static final Long CURRENT_STOCKS = 6L;
```

Comment:
This may be a bit nit-picky, but can we keep these values distinct please? That way we ensure unit tests don't mix up values. Marc used different values, but ones which are real-world (so the total for the section is correct), and then he used twice the value for the previous period